bin/tools/utils.js

```diff
@@ -5,19 +5,21 @@ const child_process = require("child_process");
 //var fsPromises = require("fs").promises;
 // node 8 compatibility
 const {promisify} = require('util');
+const stat = promisify(fs.stat);
```

Comment:
The whole file is a basket full of random utility functions with different dependencies, so this is just as it is toto make it work. 

It would be a waste of time to clean up this file because it needs to be replaced by a fully rewritten static class that is included in the "qooxdoo" (qx.server) npm package and this way can be accessed by nodeJS. As I have written elsewhere, the whole utility method business requires a rewrite, but this is outside the scope of this PR.